combustion-android-ble/src/main/java/inc/combustion/framework/ble/device/BleDevice.kt

```diff
@@ -101,14 +100,16 @@ internal open class BleDevice (
             }
         }
 
-    private val advertisementForProbe = hashMapOf<String, CombustionAdvertisingData>()
     private val remoteRssi = AtomicInteger(0)
     private val connectionMonitor = IdleMonitor()
 
     val rssi get() = remoteRssi.get()
     var connectionState = DeviceConnectionState.OUT_OF_RANGE
     val isConnected = AtomicBoolean(false)
 
+    /*
```

Comment:
Please remove this commented-out code if it's not going to be used again.

Reply:
Sorry about that -- got it cleaned up.